cmd/bisync/log.go

```diff
@@ -29,16 +29,29 @@ func (b *bisyncRun) indent(tag, file, msg string) {
 		logf = fs.Logf
 	}
 
+	if b.opt.DryRun {
+		logf = fs.Logf
+		msg += " in dry-run mode"
+	}
+
 	if tag == "Path1" {
 		tag = Color(terminal.CyanFg, "Path1")
+	} else if tag == "Path2" {
+		tag = Color(terminal.CyanFg, "Path2")
 	} else {
 		tag = Color(terminal.BlueFg, tag)
 	}
+
 	msg = Color(terminal.MagentaFg, msg)
-	msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to"), -1)
-	msg = strings.Replace(msg, "Queue delete", Color(terminal.RedFg, "Queue delete"), -1)
+	msg = strings.Replace(msg, "Queue copy to", Color(terminal.GreenFg, "Queue copy to Path2"), -1)
+	msg = strings.Replace(msg, "Queue delete", Color(terminal.RedFg, "Queue delete from Path1"), -1)
 	file = Color(terminal.CyanFg, escapePath(file, false))
-	logf(nil, "- %-18s%-43s - %s", tag, msg, file)
+
+	if b.opt.DryRun && (strings.Contains(msg, "Queue copy to Path2") || strings.Contains(msg, "Queue delete from Path1")) {
```

Comment:
I am confused about this... aren't both conditions doing the same thing? What was the intent of this change?

Reply:
Sorry, this one is a mistake. My apologies.